File: pkg/vo/verification.go
// Package vo 提供验证码相关的视图对象定义
// 创建者：Done-0
// 创建时间：2025-08-10
package vo

// SendEmailCodeResponse 发送邮箱验证码响应
type SendEmailCodeResponse struct {
	Message string `json:"message"` // 响应消息
}

File: themes/frontend/src/types/verification.ts
/**
 * 验证相关类型定义
 */

// ===== 请求类型 (Request) =====

// SendEmailCodeRequest 发送邮箱验证码请求
export interface SendEmailCodeRequest {
  email: string; // 邮箱地址
}

// ===== 响应类型 (Response) =====

// SendEmailCodeResponse 发送邮箱验证码响应
export interface SendEmailCodeResponse {
  message: string; // 响应消息
}

// ===== 客户端状态类型 =====

// VerificationState 验证状态（客户端使用）
export interface VerificationState {
  email: string;
  codeSent: boolean;
  countdown: number;
  loading: boolean;
}
